app/scripts/components/common/catalog/catalog-content.tsx

```diff
@@ -34,6 +35,9 @@ export interface CatalogContentProps {
   search: string;
   taxonomies: Record<string, string[]>;
   onAction: (action: FilterActions, value?: any) => void;
+  linkProperties: LinkProperties;
+  location?: Location | string;
+  rootPath?: string;
```

Comment:
Asking for both `location` and `rootPath` separately because location is used filterCards component and is expected from nextJs instance and container. While `rootPath` is used to decide the navigation path when clicking on a card. Container doesn't need to pass this in for now because it is defaulted but nextJs instance does